src/web/app/components/question-responses/per-question-view-responses/per-question-view-responses.component.ts

```diff
@@ -78,6 +79,9 @@ export class PerQuestionViewResponsesComponent extends InstructorResponsesViewBa
   private filterResponses(): void {
     const responsesToShow: ResponseOutput[] = [];
     for (const response of this.responses) {
+      response.recipient = StringHelper.removeAnonymousHash(response.recipient);
+      response.giver = StringHelper.removeAnonymousHash(response.giver);
+
```

Comment:
Same for this, extract out this logic into its own method instead of grouping it with `filterResponses`.